hdr_html_canvas_element.md

```diff
@@ -112,77 +114,100 @@ Update `PredefinedColorSpace` to include the following new color spaces.
 
-#### `extended-linear-srgb`
+The conversion from color space A to color space B is performed according to the following steps:
 
-To convert `extended-linear-srgb` to XYZD50 under relative colorimetric intent, perform the following steps:
+* apply the inverse transfer function of color space A
+* convert to the connection space by multiplying by the connection matrix of color space A
+* convert to color space B by multiplying by by the inverse of the connection matrix of color space B
+* apply the transfer function of color space B
 
-* Apply the matrix transformation to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the XYZ primaries.
-* Apply the matrix transformation to convert the [sRGB white point](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the D50 white point.
+The domain and range of this conversion consist of all real values and the component signals in the connection color space are real numbers.
 
-Note that the domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
+#### `extended-linear-srgb`
+
+* Transfer function: identity
+
+* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
 
 #### `extended-srgb`
 
-To convert `extended-srgb` to XYZD50 under relative colorimetric intent, perform the following steps:
+* Transfer function: See [extended-linear-srgb](#extended-linear-srgb)
+
+* Connection matrix: matrix to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the primary colours specified in Rec. ITU-R BT.2100
 
-* Convert each color channel to linear space.
-  * For each color channel value `x`, this means applying the following function:
-    * If `x < -0.4045`, then return `-pow((-x + 0.055)/1.055, 2.4)`
-    * Else if `x <= 0.4045` then return `x / 12.92`
-    * Else return `pow((x + 0.055)/1.055, 2.4)`
-* Apply the matrix transformation to convert the [sRGB primaries](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the XYZ primaries.
-* Apply the matrix transformation to convert the [sRGB white point](https://www.w3.org/TR/css-color-4/#valdef-color-srgb) to the D50 white point.
+_Note:_ The domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
 
-Note that the domain of this transformation function is all real values. Its domain is not restricted to the unit interval [0, 1].
 Also see note in the Issues section at the bottom about whether this space should be distinct from the existing `'srgb'` space.
 
 #### `rec2100-hlg`
 
-To convert `rec2100-hlg` to XYZD50 under relative colorimetric intent, perform the following steps:
+* Transfer function: HLG Reference OETF specified at Rec. ITU-R BT.2100
```

Comment:
I think this should read the "inverse of the HLG Reference OETF".

(we should probably just write out the math)

Reply:
The first step of the conversion is _apply the inverse transfer function of color space A_, i.e. linearize the color components. _HLG Reference OETF_ takes linear components and generates a non-linear components, so the _transfer function_ is _HLG Reference OETF_, right?

> (we should probably just write out the math)

Yes, it would be good to add as an informative section.